src/js/components/Settings/SettingsProfilePicture.jsx

```diff
@@ -49,6 +50,27 @@ class SettingsProfilePicture extends Component {
     if (voterPhotoQueuedToSaveSet || profileImageTypeCurrentlyActive) {
       VoterActions.voterPhotoSave(voterPhotoQueuedToSave, voterPhotoQueuedToSaveSet, profileImageTypeCurrentlyActive);
       VoterActions.voterPhotoQueuedToSave(undefined);
+
+      // Adding event data to dataLayer for Google Tag Manager
+      const currentPathname = window.location.pathname;
+      const page = lookupPageNameAndPageTypeDict(currentPathname);
+      TagManager.dataLayer({
+        dataLayer: {
+          event: 'save_profile_photo',
+          actionDetails: {
+            actionType: 'save',
+            buttonId: 'saveEditYourPhotoBottom',
+          },
+          userDetails: {
```

Comment:
Please change this to Dale's new function.

userDetails: VoterStore.getAnalyticsUserDetails(),